storage/src/main/java/com/tencent/rss/storage/handler/api/ClientReadHandler.java

```diff
@@ -18,11 +18,23 @@
 
 package com.tencent.rss.storage.handler.api;
 
+import com.tencent.rss.common.BufferSegment;
 import com.tencent.rss.common.ShuffleDataResult;
 
 public interface ClientReadHandler {
 
   ShuffleDataResult readShuffleData();
 
   void close();
+
+  // The handler only returns the segment,
+  // but does not know the actually consumed blocks,
+  // so the consumer should feedback this information.
+  // Each type of handler can design their rules.
+  void feedbackConsumedBlock(BufferSegment bs);
+
+  // Display the debug information
+  void reportConsumedBlockInfo();
```

Comment:
debug information?Why do we use LogInfo insteadof LogDebug?

Reply:
fixed